pom.xml

```diff
@@ -1445,6 +1445,15 @@
         </dependencies>
 
     </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
```

Comment:
Shouldn't this go under dependencyManagement section?

Reply:
@pulasthi it is already there in "dependencyManagement". What this does is to make available the annotation processor on the classpath for the java compiler.
Adding here makes it available for all modules(which prevents duplicates).